src/main/java/utils/ApiHelper.java

```diff
@@ -0,0 +1,95 @@
+package utils;
+
+import api.ProductCode;
+import api.ProductQuantity;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.restassured.RestAssured;
+import io.restassured.module.jsv.JsonSchemaValidator;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import static org.hamcrest.Matchers.equalTo;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ApiHelper {
+    private RequestSpecification requestSpecification;
+    public Map<String, Object> storage = new HashMap<>();
+
+    public void createCart(String url) {
+        requestSpecification = RestAssured.given()
+                .baseUri(url)
+                .basePath("/api/v2/kvn/users/anonymous/carts/");
+
+        Response response = RestAssured.given()
+                .spec(requestSpecification)
```

Comment:
once you setting the global RequestSpecification you wouldn't need to specify it in each api call